base/complex.jl

```diff
@@ -541,6 +541,30 @@ function cis(z::Complex)
     Complex(v * c, v * s)
 end
 
+cispi(theta::Real) = Complex(reverse(sincospi(theta))...)
+
+"""
+    cispi(z)
+
+Return ``\\exp(iπz)``.
```

Comment:
I think it would be a bit more informative to copy the `sinpi` docstring here:
```suggestion
Compute ``\\exp(i\\pi x)`` more accurately than `cis(pi*x)`, especially for large `x`.
```